gmail/modules/listener/Dispatcher.bal

```diff
@@ -64,9 +64,7 @@ class Dispatcher {
         }
     }
 
-    isolated function dispatch(gmail:MailboxHistoryPage mailboxHistoryPage) returns @tainted error? {
-        if (mailboxHistoryPage?.history is gmail:History[]) {
-            foreach var history in <gmail:History[]>mailboxHistoryPage?.history {
+    isolated function dispatch(gmail:History history) returns @tainted error? {
```

Comment:
Does this mean any event we get for gmail listener will be of type `gmail:History`?

Reply:
Yes, Actually what ever change happens, listener will get a history update, using this history records, we need to find what change happened in Mailbox.